packages/block-editor/src/store/selectors.js

```diff
@@ -1325,7 +1325,7 @@ export const getInserterItems = createSelector(
 			return {
 				id,
 				name: blockType.name,
-				initialAttributes: {},
+				initialAttributes: blockType.initialAttributes || {},
```

Comment:
@youknowriad would like to confirm it's not against the web and will not break anything.

Reply:
Yup, It'd be great to confirm that.

Reply:
Why would a block type define "initialAttributes"? I think that's the role of the block variations.